src/test/regress/expected/distributed_functions.out

```diff
@@ -470,14 +470,16 @@ SELECT * FROM run_command_on_workers($$SELECT function_tests.eq('012345689ab','0
  localhost |    57638 | t       | f
 (2 rows)
 
+--
+-- Test distributed functions of extension.
+--
 -- distributed functions should not be allowed to depend on an extension, also functions
 -- that depend on an extension should not be allowed to be distributed.
 ALTER FUNCTION eq(macaddr,macaddr) DEPENDS ON EXTENSION citus;
 ERROR:  distrtibuted functions are not allowed to depend on an extension
 DETAIL:  Function "function_tests.eq(pg_catalog.macaddr,pg_catalog.macaddr)" is already distributed. Functions from extensions are expected to be created on the workers by the extension they depend on.
 SELECT create_distributed_function('pg_catalog.citus_drop_trigger()');
-ERROR:  unable to create a distributed function from functions owned by an extension
-DETAIL:  Function "pg_catalog.citus_drop_trigger()" has a dependency on extension "citus". Functions depending on an extension cannot be distributed. Create the function by creating the extension on the workers.
+ERROR:  Extension(citus) owning the function(citus_drop_trigger) is not distributed
```

Comment:
There are actually good reasons to prevent citus functions to be marked as distributed, we should have a separate case for that in our code above, especially if we succeed in automatically marking the extension distributed.

Reply:
ERROR:  Citus extension functions(citus_drop_trigger) can't be distributed.